extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/ONE.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sonos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Player ONE Thing Type -->
+	<thing-type id="One" listed="false">
+		<!-- Label without column (:) because of device pairing name restrictions -->
+		<label>ONE</label>
+		<description>Represents SONOS ONE Zone Player</description>
+
+		<channels>
+			<channel id="add" typeId="add" />
+			<channel id="alarm" typeId="alarm" />
+			<channel id="alarmproperties" typeId="alarmproperties" />
+			<channel id="alarmrunning" typeId="alarmrunning" />
+			<channel id="control" typeId="control" />
+			<channel id="currentalbum" typeId="currentalbum" />
+			<channel id="currentalbumart" typeId="currentalbumart" />
+			<channel id="currentalbumarturl" typeId="currentalbumarturl" />
+			<channel id="currentartist" typeId="currentartist" />
+			<channel id="currenttitle" typeId="currenttitle" />
+			<channel id="currenttrack" typeId="currenttrack" />
+			<channel id="shuffle" typeId="shuffle" />
+			<channel id="repeat" typeId="repeat" />
+			<channel id="favorite" typeId="favorite" />
+			<channel id="led" typeId="led" />
+			<channel id="localcoordinator" typeId="localcoordinator" />
+			<channel id="mute" typeId="mute" />
+			<channel id="notificationsound" typeId="notificationsound" />
+			<channel id="notificationvolume" typeId="notificationvolume" />
+			<channel id="playlist" typeId="playlist" />
+			<channel id="clearqueue" typeId="clearqueue" />
+			<channel id="playqueue" typeId="playqueue" />
+			<channel id="playtrack" typeId="playtrack" />
+			<channel id="playuri" typeId="playuri" />
+			<channel id="publicaddress" typeId="publicaddress" />
+			<channel id="radio" typeId="radio" />
+			<channel id="remove" typeId="remove" />
+			<channel id="restore" typeId="restore" />
+			<channel id="restoreall" typeId="restoreall" />
+			<channel id="save" typeId="save" />
+			<channel id="saveall" typeId="saveall" />
+			<channel id="snooze" typeId="snooze" />
+			<channel id="standalone" typeId="standalone" />
+			<channel id="state" typeId="state" />
+			<channel id="stop" typeId="stop" />
+			<channel id="tuneinstationid" typeId="tuneinstationid" />
+			<channel id="volume" typeId="volume" />
+			<channel id="zonegroup" typeId="zonegroup" />
+			<channel id="zonegroupid" typeId="zonegroupid" />
+			<channel id="zonename" typeId="zonename" />
+			<channel id="coordinator" typeId="coordinator" />
+			<channel id="sleeptimer" typeId="sleeptimer" />
+			<channel id="currenttransporturi" typeId="currenttransporturi" />
+			<channel id="currenttrackuri" typeId="currenttrackuri" />
+		</channels>
```

Comment:
Is there no line-in in the One ? If line-in is present, take example on the PLAY:5.